tools/EventClients/Clients/KodiSend/kodi-send.py

```
@@ -74,6 +82,14 @@ def main():
             keymap = a
         elif o == "--button":
             actions.append({'type': TYPE_BUTTON, 'content': a})
+        elif o == "--mouse":
+            actions.append({'type': TYPE_MOUSE, 'content': a})
+        elif o == "--log":
+            actions.append({'type': TYPE_LOG, 'content': a})
+        elif o == "--loglevel":
+            loglevel = int(a)
```

Check for TypeError or just let it raise?